js/api.md

```diff
@@ -362,6 +362,28 @@ Amplify.configure({
 });
 ```
 
+#### AWS AppSync Multi-Auth
+
+AWS AppSync can support multiple authentication modes on a single API. [LINK TO APPSYNC DOCS](http://docs). In order to use this feature with the Amplify Graphql Client the `API.graphql({...})` function would accept an optional parameter called `authMode`, its value will be one of the supported auth modes:
+
+- `API_KEY`
+- `AWS_IAM`
+- `OPENID_CONNECT`
+- `AMAZON_COGNITO_USER_POOLS`
+
+If the authMode is not provided, Amplify will use the default auth mode that was configured:
```

Comment:
Where was this configured? In the client constructor? The way this is written it reads as if you're showing how to configure the default mode however the sample code is showing how to override on the method invocation.